lib/action_network_rest/base.rb

```diff
@@ -17,6 +17,19 @@ def list(page: 1)
       objects
     end
 
+    def all
```

Comment:
It seems like you can easily go over the [rate limit](https://actionnetwork.org/docs/#considerations) while using this method. May be worth adding some logic for retying with a slight delay if we reach it

Reply:
Added to this method so it now limits the rate is calls `list` to every 0.25 secs max (as the docs mention). It also exponentially backs off if the 429 (Too Many Requests) response code is received. I think this should do it.

Worth noting this makes running `bundle exec rake -t` a little slower because it test the back off is working okay.